firebase.json

```diff
@@ -151,9 +163,7 @@
       { "source": "/go/android-embedding-move", "destination": "https://docs.google.com/document/d/1nQujwZfEe3QOHTyZn160eImpoJOYioADP09DtumcLcc/edit?ts=5d8041db#", "type": 301 },
       { "source": "/go/android-migration-summary", "destination": "https://docs.google.com/document/d/1wKspwf6LQu6uo32uQ9NcukfiKhLL4ButV9cwpjeb8QI", "type": 301 },
       { "source": "/go/android-ndk-version", "destination": "https://docs.google.com/document/d/1sljfqHpI5m4uxsL7B89yhYluO97Mz1YKyA0VgzzXyZ4/edit?usp=sharing&resourcekey=0-Ts9Rk8jbJor7WBMMiehKWg", "type": 301 },
-      { "source": "/go/android-plugin-migration.", "destination": "/development/packages-and-plugins/plugin-api-migration", "type": 301 },
-      { "source": "/go/android-plugin-migration", "destination": "/development/packages-and-plugins/plugin-api-migration", "type": 301 },
-      { "source": "/go/android-project-migration", "destination": "https://github.com/flutter/flutter/wiki/Upgrading-pre-1.12-Android-projects", "type": 301 },
```

Comment:
This link was completely removed here. I don't think we want to do that. See b/241819690.